docs/Using-EthSigner/Getting-Started.md

```diff
@@ -31,6 +31,16 @@ option set to `8590`.
 
 Create a text file containing the password for the V3 Keystore key file to be created (for example, `passwordFile`). 
 
+!!! attention "The password text file must not contain end-of-line character"
```

Comment:
The password file should not contain ANY characters not in your password - whitespace, EOL .... what if the end-of-line is part of your password?

Maybe state that its treated as a binary file.